### PR TITLE
Add Okta import rule to Terraform.

### PR DESCRIPTION
**Pull request by mdwn**
The Terraform provider now allows specifying Okta import rules, which should make automation of Okta import rules simpler.

Depends on:
- https://github.com/gravitational/protoc-gen-terraform/pull/29

### CODE REVIEW

**Review by r0mant — APPROVED** [review]
@marcoandredinis Can you please review this as well?

File: terraform/protoc-gen-terraform-teleport.yaml [hunk 1]
```diff
@@ -126,16 +134,14 @@ exclude_fields:
...
+    - "SessionRecordingConfigV2.Metadata.Name" # It's a singleton resource
 
     # User
     - "UserSpecV2.LocalAuth" # We do not need to set user auth data
     - "UserSpecV2.Expires"   # Those fields are Teleport side meta, we do not need to change them via Terraform
     - "UserSpecV2.CreatedBy"
     - "UserSpecV2.Status"
 
-name_overrides:
-
```
Comment (marcoandredinis) [inline terraform/protoc-gen-terraform-teleport.yaml:138]: I know we are not using it but can we keep it here? Just to create awareness that this feature exists
It's useful for handling field renames

**Comment by marcoandredinis** [comment]
I think we have to update the teleport version we use for testing, the current one doesn't seem to have the Okta Services.
It's currently using 12.1
https://github.com/gravitational/teleport-plugins/issues/820

File: terraform/protoc-gen-terraform-teleport.yaml (hunk 1 above)
Comment (mdwn) [reply terraform/protoc-gen-terraform-teleport.yaml:138]: Oh sure, I actually did not intend to delete this!

**Review by marcoandredinis — APPROVED** [review]

**Pull request merged**